ember-cli-build.js

```diff
@@ -18,29 +22,22 @@ module.exports = function (defaults) {
         },
       },
     },
-    '@lblod/ember-vo-webuniversum': {
-      'shouldImportComponentCss': false,
-    },
     sassOptions: {
-      sourceMapEmbed: true,
-      includePaths: [
-        'node_modules/@appuniversum/appuniversum',
-        'node_modules/@appuniversum/ember-appuniversum/app/styles',
-        'node_modules/@lblod/ember-rdfa-editor/app/styles'
-      ]
+      sourceMap: !envIsProduction,
+      sourceMapEmbed: !envIsProduction,
     },
     autoprefixer: {
       enabled: true,
       cascade: true,
-      sourcemap: true
+      sourcemap: !envIsProduction
     },
     flatpickr: {
```

Comment:
I guess flatpickr config can go now as well

Reply:
Updated the citaten-plugin version with AppUniversum component.